Practical8/Practical8/ManageUserThings.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practical8
+{
+    public class ManageUserThings : IManageUserThings
+    {
+        public enum Options
+        {
+            Add = 1,
+            Update = 2,
+            ViewStudentByID = 3,
+            ViewAllStudents = 4,
+            DeleteStudent = 5,
+            Exit = 6
+        }
+        /// <summary>
+        /// This method will give different options and based on the 
+        /// option selection we can do different CRUD operation.
+        /// </summary>
+        /// <param name="studentList"></param>
+        public void ManageUsers(IManageStudentList studentList)
+        {
+            while (true)
+            {
+
+
+                Console.WriteLine("Choose Options : \n1.Add Student \n2.Update Student Details\n3.View Student By ID " +
+                    "\n4.View All Students \n5.Delete Student \n6.Exit");
+                int option;
+                try
+                {
+                    option = int.Parse(Console.ReadLine());
```

Comment:
you can give a check like user has entered a wrong value if user enters something else